src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2400,10 +2400,16 @@ internal void AddInterception(string filePath, int line, int character, Location
 
         #region Binding
 
+        public new SemanticModel GetSemanticModel(SyntaxTree syntaxTree, bool ignoreAccessibility)
+#pragma warning disable RSEXPERIMENTAL001 // Internal usage of experimental API
+            => GetSemanticModel(syntaxTree, ignoreAccessibility ? SemanticModelOptions.IgnoreAccessibility : SemanticModelOptions.None);
+#pragma warning restore RSEXPERIMENTAL001
+
         /// <summary>
         /// Gets a new SyntaxTreeSemanticModel for the specified syntax tree.
         /// </summary>
-        public new SemanticModel GetSemanticModel(SyntaxTree syntaxTree, bool ignoreAccessibility)
+        [Experimental("RSEXPERIMENTAL001")]
```

Comment:
Let's extract this code to a well-known file so we can keep track of current error codes.